### PR TITLE
Reuse bill selector component to commission and cash out screens

### PR DESCRIPTION
**Pull request by chaotixkilla**

### CODE REVIEW

**Comment by chaotixkilla** [comment]
[Implement denomination picker on Minimum TX and Cash Out panels](https://trello.com/c/bAKl1kwB/657-implement-denomination-picker-on-minimum-tx-and-cash-out-panels)

**Review by hellypi — APPROVED** [review]
Looks good

File: new-lamassu-admin/src/pages/Commissions/components/CommissionsDetails.js [hunk 1]
```diff
@@ -6,15 +6,15 @@ import Section from 'src/components/layout/Section'
...
+  getSchema,
   getOverridesSchema,
   defaults,
   overridesDefaults,
   getOrder
 } from 'src/pages/Commissions/helper'
 
 const CommissionsDetails = memo(
-  ({ config, currency, data, error, save, saveOverrides }) => {
+  ({ config, locale, currency, data, error, save, saveOverrides, classes }) => {
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/components/CommissionsDetails.js]: I'm confused over this classes here, doesn't seem to be used.

File: new-lamassu-admin/src/pages/Commissions/Commissions.js [hunk 2]
```diff
@@ -16,6 +16,13 @@ import CommissionsList from './components/CommissionsList'
...
   listViewButton: {
     marginLeft: 4
+  },
+  autoComplete: {
+    width: '100%'
+  },
+  bold: {
+    fontWeight: 'bold',
+    height: 200
   }
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/pages/Commissions/Commissions.js]: Don't seem to be used

File: new-lamassu-admin/src/utils/bill-options.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import * as R from 'ramda'
+
+const getBillOptions = R.curry((locale, denomiations) => {
+  const currency = R.prop('fiatCurrency')(locale)
+  return R.compose(
+    R.map(code => ({ code: parseInt(code), display: code })),
+    R.keys,
+    R.path([currency])
```
Comment (RafaelTaranto) [inline new-lamassu-admin/src/utils/bill-options.js:8]: I don't think ramda is null safe, does this work even when picking a fiat that has no bill definition?

**Review by RafaelTaranto — CHANGES_REQUESTED** [review]

File: new-lamassu-admin/src/utils/bill-options.js (hunk 3 above)
Comment (chaotixkilla) [reply new-lamassu-admin/src/utils/bill-options.js:8]: I tested with USD and COP, with and without bill definitions, respectively, and everything seems to work correctly. When there's no bill definitions, it returns an empty array and allows the user to insert a number

**Review by RafaelTaranto — APPROVED** [review]

**Pull request merged**